### PR TITLE
Update .travis.yml

### LINKED ISSUE #89: Add test cases for python3.7 and python3.8
Currently both `tox.ini` and `travis.yml` do not support latest python versions.

I can send a PR with the fix! 

Related: https://github.com/wemake-services/wemake-python-styleguide/issues/1138

### PR DESCRIPTION
**Pull request by sobolevn**
Closes #89

### CODE REVIEW

**Comment by sobolevn** [comment]
@zheller any ideas why the tests are failing?

**Comment by twolfson** [comment]
Neither of us have maintained this project in a while so @sobolevn, your guess is as good as ours -- especially since you seem to know more about what's changed in Python 3.7. Do you mind taking a look?

Travis CI failures can be seen by clicking on the link provided by GitHub. Here's the 3.7 failure:

https://travis-ci.org/zheller/flake8-quotes/jobs/660032029

```
Traceback (most recent call last):

  File "/home/travis/build/zheller/flake8-quotes/test/test_checks.py", line 23, in test_stdin

    self.assertEqual(stderr, b'')

AssertionError: b"/home/travis/virtualenv/python3.7.1/lib/[155 chars]')\n" != b''
```

**Comment by sobolevn** [comment]
Sure, I will refactor the tests today 👍

**Comment by twolfson** [comment]
Unsure that refactoring is necessary ._. Maybe triage first and report back so you don't waste time with a refactor?

**Comment by skirpichev** [comment]
@twolfson, are you ok with the flake8 requirement change?

**Comment by twolfson** [comment]
Ah, didn't realize this PR was ready for review. Looking over now

File: requirements-dev.txt [hunk 1]
```diff
@@ -1 +1 @@
-flake8<3.4.0,>=3.3.0
+flake8>=3.5
```
Comment (twolfson) [inline requirements-dev.txt]: I'm fine with making our requirement float more but can we lock this down to `<4.0`? Usually a major revision bump means breaking changes =/

```suggestion
flake8<4.0,>=3.5
```

File: setup.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def read(*filenames, **kwargs):
     author_email='zheller@gmail.com',
     version=about['__version__'],
     install_requires=[
-        'flake8',
+        'flake8>=3.5',
```
Comment (twolfson) [inline setup.py]: Can you explain this change a little more?

Comment (sobolevn) [reply setup.py]: For some reason resolved `flake8` version was 3.2
It didn't work with `3.8`

The first version of `flake8` which worked is `3.5`

Comment (twolfson) [reply setup.py]: Weird, I'd expect the `requirements-dev.txt` to supercede this. It looks like in fact, that is what was happening and this change wasn't respected in Travis CI:

(I think you mean `flake8@3.3`, not `3.2` for Python 3.8)

https://github.com/zheller/flake8-quotes/pull/90/commits/750f4a0f6bda934014970d2f48638c2417eb3f52

https://travis-ci.org/github/zheller/flake8-quotes/jobs/660092929#L191

The following change where you updated `requirements-dev.txt` installs appropriately

https://travis-ci.org/github/zheller/flake8-quotes/jobs/660095678#L191

https://github.com/zheller/flake8-quotes/pull/90/commits/795cbac666bf3c34dd54b0c63459ae543f8f763d

Can you rollback this line change?

Comment (twolfson) [reply setup.py]: Looks like I can edit the file in the PR directly so going to do that for speedier turnaround

**Comment by twolfson** [comment]
Looks like proper version from `requirements-dev.txt` installed correctly even after rollback :+1:

https://travis-ci.org/github/zheller/flake8-quotes/jobs/666956750#L190

Landing now

**Comment by twolfson** [comment]
Fwiw, nothing public facing changed so only landing this as a patch revision)

**Comment by twolfson** [comment]
Released in `2.1.2`. Thanks for the PR =)

**Comment by sobolevn** [comment]
Thanks a lot! 👍

**Pull request merged**